web3/_utils/module_testing/shh_module.py

```diff
@@ -385,6 +395,8 @@ def test_shh_post(self, web3):
             "ttl": 400,
         })
 
+        assert response is True
```

Comment:
this change looks a little suspect to me, but probably fine

Reply:
oops, don't need that 😆